iina/MainWindowController.swift

```diff
@@ -57,16 +55,11 @@ class MainWindowController: NSWindowController, NSWindowDelegate {
 
   // MARK: - Objects, Views
 
-  unowned var player: PlayerCore
-
-  lazy var videoView: VideoView = {
-    let view = VideoView(frame: self.window!.contentView!.bounds)
-    view.player = self.player
-    return view
-  }()
-
-  /** A responder handling general menu actions. */
-  var menuActionHandler: MainMenuActionHandler!
+  override var videoView: VideoView {
```

Comment:
We may want to consider making this kind of thing a property wrapper at some point. I think the proposal has [a decent example](https://github.com/apple/swift-evolution/blob/master/proposals/0258-property-wrappers.md#proposed-solution) of what this can look like.

Reply:
Wow, very cool stuff. We can maybe rewrite a lot of code using property wrapper.